### ISSUE TITLE
Support for emotes in other places (Kick messages, signs, etc)

### ISSUE THREAD

**Issue by LittleBigBug** [XspeedPL/Streamotes#11]
Would be nice to have support for emotes in places like signs, kick messages, on-screen titles, boss bar, etc

**Comment by Petrol12** [comment]
I agree with that, it would be very cool. Although it can be problematic. I am playing on a server where there is a set of 7tv emojis, unfortunately, it is limited. But they are displayed on signage.
